packages/mdc-slider/index.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import MDCComponent from '@material/base/component';
+
+import {strings} from './constants';
+import MDCSliderFoundation from './foundation';
+
+export {MDCSliderFoundation};
+
+export class MDCSlider extends MDCComponent {
+  static attachTo(root) {
+    return new MDCSlider(root);
+  }
+
+  get value() {
+    return this.foundation_.getValue();
+  }
+
+  set value(value) {
+    this.foundation_.setValue(value);
+  }
+
+  get min() {
+    return this.foundation_.getMin();
+  }
+
+  set min(min) {
+    this.foundation_.setMin(min);
+  }
+
+  get max() {
+    return this.foundation_.getMax();
+  }
+
+  set max(max) {
+    this.foundation_.setMax(max);
+  }
+
+  get step() {
+    return this.foundation_.getStep();
+  }
+
+  set step(step) {
+    this.foundation_.setStep(step);
+  }
+
+  get disabled() {
+    return this.foundation_.isDisabled();
+  }
+
+  set disabled(disabled) {
+    this.foundation_.setDisabled(disabled);
+  }
+
+  initialize() {
+    this.thumbContainer_ = this.root_.querySelector(strings.THUMB_CONTAINER_SELECTOR);
+    this.track_ = this.root_.querySelector(strings.TRACK_SELECTOR);
+  }
+
+  getDefaultFoundation() {
+    return new MDCSliderFoundation({
+      addClass: (className) => this.root_.classList.add(className),
+      removeClass: (className) => this.root_.classList.remove(className),
+      getAttribute: (name) => this.root_.getAttribute(name),
+      setAttribute: (name, value) => this.root_.setAttribute(name, value),
+      removeAttribute: (name) => this.root_.removeAttribute(name),
+      computeBoundingRect: () => this.root_.getBoundingClientRect(),
+      getTabIndex: () => this.root_.tabIndex,
```

Comment:
+1 to @traviskaufman .I think it is also how [MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/tabIndex) recommend for accessing tabIndex.